txcache/common.go

```diff
@@ -0,0 +1,5 @@
+package txcache
+
+import "github.com/multiversx/mx-chain-go/common/holders"
+
+var defaultBlockchainInfo = holders.NewBlockchainInfo(nil, 0)
```

Comment:
Is this for test code only? Then, maybe move it to the file that holds test utilities?